app/uploaders/league_header_image_uploader.rb

```diff
@@ -5,12 +5,7 @@ def store_dir
     "uploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
   end
 
-  # process :scale => [800, 300]
-  # process resize_to_fit: [800, 300]
-
-  # def scale(width, height)
-  #   # do something
-  # end
+  process resize_to_fit: [1920, 1920]
```

Comment:
I'd recommend to save the original and create different versions with different sizes for retina and non-retina. Doing this, you'll always be able to recreate different versions from your original file.